File: solutions/3555-final-array-state-after-k-multiplication-operations-i/solution.js
/**
 * @param {number[]} nums
 * @param {number} k
 * @param {number} multiplier
 * @return {number[]}
 */
var getFinalState = function(nums, k, multiplier) {
    
    for(let i = 0; i < k; i++){
        let min = Math.min(...nums);
        let i = nums.indexOf(min);
        
        nums[i] = nums[i] * multiplier;
    }
    
    
    return nums;
};

File: solutions/3555-final-array-state-after-k-multiplication-operations-i/solution.py
class Solution:
    def getFinalState(self, nums: List[int], k: int, multiplier: int) -> List[int]:
        for x in range(0,k):
            min = None
            idx = -1
            for i, num in enumerate(nums):
                if min is None or num < min:
                    min = num
                    idx = i
            
            nums[idx] = nums[idx] * multiplier
        
        return nums
